_unused_topics/cluster-logging-configuring-node-selector.adoc

```diff
@@ -45,20 +45,13 @@ spec:
       nodeSelector:  <2>
         logging: kibana
       replicas: 1
-  curation:
```

Comment:
note to myself: perhaps curator should be removed from CLO CSV as well

https://github.com/openshift/cluster-logging-operator/blob/master/manifests/5.1/cluster-logging.v5.1.0.clusterserviceversion.yaml#L326-L327
https://github.com/openshift/cluster-logging-operator/blob/master/manifests/5.1/cluster-logging.v5.1.0.clusterserviceversion.yaml#L404-L416
Cc: @jcantrill